src/components/views/messages/RoomCreate.js

```diff
@@ -53,7 +53,7 @@ module.exports = React.createClass({
         permalinkCreator.load();
         const predecessorPermalink = permalinkCreator.forEvent(predecessor['event_id']);
         return <div className="mx_CreateEvent">
-            <img className="mx_CreateEvent_image" src={require("../../../../res/img/room-continuation.svg")} />
+            <img className="mx_CreateEvent_image" />
```

Comment:
Probably it should just be a `div` or something since it no longer loads an image itself?